### ISSUE TITLE
Setup of Pages unclear

### ISSUE THREAD

**Issue by IoannesCarolus** [fhorinek/BB#141]
Dear all,
I'm running fw208.
I tried to create an additional page as 'Thermal Assistant'.

- I can create 2 different layouts with widgets,

but...

- When changing the name for this 1 page, the names are changed for both existing pages.

- When setting 'Page autoset' 'switch to page...' » 'Circling' then this gets applied for both existing pages. 

How is it possible to create independent coexisting pages in the Strato?

How can I manage then 2 different pages?

BR J.C.

**Comment by fhorinek** [comment]
I think that you found a bug, can you send me zipped config directory to fero@skybean.eu

**Comment by IoannesCarolus** [comment]
Just send the cpl zipped config dir via email

**Comment by IoannesCarolus** [comment]
I tested with a fully wiped strato directly flashed with fw208.

- If you jump into a page via <left or >right hard-button while on the main screen, and then go into "edit page" then the correct name is shown, the same also with the "Autoset".
- With "Move left" or "Move right" in the "Page settings" the arrangement is managed.
- **Trigger**: The user needs always to first select the page he/she wants to edit AND then go to edit.

Bottom line:
From the dev's intention this feature works but the user experience is not clear towards Page Settings/Page Management.

@fhorinek: Please include a proper description of Pages Settings/Creation/Management into your manual/docu on https://docs.google.com/document/d/1UVKun-YTFogi0B5zxckbwWZP4xeWZfG2w7wkskMC3cc/edit
so other users benefit from it.
